plugins/onex/skills/ci-watch/SKILL.md

```
@@ -145,20 +146,77 @@ If neither inbox-wait nor `_bin/ci-status.sh` is available, use `gh run watch` d
+     BASE_FAILING=$(gh run view "$BASE_RUN_ID" --repo {repo} --json jobs \
+       -q '[.jobs[] | select(.conclusion == "failure") | .name]')
+     ```
+
+     - **Pre-existing**: check name appears in `BASE_FAILING` (failed on base branch before this PR)
+     - **Introduced**: failure is new — only present on this PR's branch
+
+   - For each **pre-existing** failure:
+     - Log: `"Pre-existing CI failure: {check_name} — dispatching separate fix PR (not blocking current PR)"`
+     - Dispatch a **background** fix agent targeting `main` (not this branch):
+
+       ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)